lib/rules/no-positional-data-test-selectors.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const Rule = require('./base');
+const { builders: b } = require('ember-template-recast');
+
+const BUILT_INS = [
+  'action',
+  'array',
+  'component',
+  'concat',
+  'debugger',
+  'each',
+  'each-in',
+  'fn',
+  'get',
+  'hasBlock',
+  'has-block',
+  'has-block-params',
+  'hash',
+  'if',
+  'input',
+  'let',
+  'link-to',
+  'loc',
+  'log',
+  'mount',
+  'mut',
+  'on',
+  'outlet',
+  'partial',
+  'query-params',
+  'textarea',
+  'unbound',
+  'unless',
+  'with',
+  '-in-element',
+  'in-element',
+];
+
+module.exports = class NoPositionalDataTestSelectors extends Rule {
+  visitor() {
+    return {
+      MustacheStatement(node) {
+        this.process(node);
+      },
+      BlockStatement(node) {
+        this.process(node);
+      },
+    };
+  }
+
+  process(node) {
+    if (BUILT_INS.includes(node.path.original)) {
+      return;
+    }
```

Comment:
I'm curious, why do we have to avoid built-ins? Is this to support that `if` case I mentioned adding?

Reply:
yes @rwjblue